dask_kubernetes/core.py

```diff
@@ -358,7 +358,7 @@ def scale(self, n):
         """
         pods = self._cleanup_terminated_pods(self.pods())
         if n >= len(pods):
-            return self.scale_up(n, pods=pods)
```

Comment:
uh oh! This change ended up turning off scale_up all together. I've opened a new PR to fix this: #150